packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1576,7 +1576,7 @@ export default class NewVm extends BaseComponent {
         </label>
       ) : null
 
-    const isVtpmSupported = pool.vtpmSupported
+    const isVtpmSupported = pool?.vtpmSupported ?? true
```

Comment:
Do we really want to assume VTPM is supported if we don't know?

Reply:
IMO, yes. Otherwise, this means we will not allow a user to create a VM with a VTPM on a pool that supports it.

Reply:
But if the user tries to create a VM with a VTPM but it's not supported, it will fail and they'll have to try again, right? Is the error message clear when VTPM isn't supported?

Reply:
The error thrown is:
```js
// @xen-orchestra/xapi/vtpm.mjs#10
 if (pool.restrictions.restrict_vtpm !== 'false') {
      throw incorrectState({
        actual: pool.restrictions.restrict_vtpm,
        expected: 'false',
        object: pool.uuid,
        property: 'restrictions.restrict_vtpm',
      })
    }

Reply:
And this error is shown in a toaster? What I want to avoid is a user trying to create a VM and not understanding immediately why it's not working.

Reply:
Hm, the error is displayed in a toaster, but it only displays: `incorrect state`.
So, I think we need to catch this error and show a clearer error in the toaster.

Reply:
Yes, I think it's necessary since it's an expected error (you only need a pool that doesn't support VTPM and a user that doesn't have permissions on the pool to trigger the error), so a simple catch for that exact error and an error toaster with a clear message is enough I think.